src/legacy/core_plugins/vis_type_tagcloud/public/components/tag_cloud_visualization.js

```diff
@@ -31,130 +31,132 @@ import { FeedbackMessage } from './feedback_message';
 
 const MAX_TAG_COUNT = 200;
 
-export class TagCloudVisualization {
-  constructor(node, vis) {
-    this._containerNode = node;
-
-    const cloudRelativeContainer = document.createElement('div');
-    cloudRelativeContainer.classList.add('tgcVis');
-    cloudRelativeContainer.setAttribute('style', 'position: relative');
-    const cloudContainer = document.createElement('div');
-    cloudContainer.classList.add('tgcVis');
-    cloudContainer.setAttribute('data-test-subj', 'tagCloudVisualization');
-    this._containerNode.classList.add('visChart--vertical');
-    cloudRelativeContainer.appendChild(cloudContainer);
-    this._containerNode.appendChild(cloudRelativeContainer);
-
-    this._vis = vis;
-    this._truncated = false;
-    this._tagCloud = new TagCloud(cloudContainer);
-    this._tagCloud.on('select', event => {
-      if (!this._visParams.bucket) {
-        return;
-      }
-      this._vis.API.events.filter({
-        table: event.meta.data,
-        column: 0,
-        row: event.meta.rowIndex,
+export function createTagCloudVisualization({ colors }) {
```

Comment:
We should be able to revert this because colors can just be imported, right? Same for the test fields above.